server/src/main/java/io/spine/server/procman/LifecycleRules.java

```diff
@@ -63,8 +63,12 @@ public LifecycleRules() {
     /**
      * Configures the repository to archive its entities when certain events/rejections occur.
      *
-     * <p>Subsequent calls to this method do not clear the previously added event classes (i.e. not
-     * override each other).
+     * <p>Subsequent calls to this method do not clear the previously added event classes,
+     * i.e, in order not to override each other. For example, if we make two subsequent calls:
+     * `lifecycle.archiveOn(Event1.class)`;
```

Comment:
It's probably better to use the `{@code}` tag like this: `{@code lifecycle.archiveOn(Event1.class);}`, so it's properly formatted in the IDE tool-tip.